folly/Portability.h

```diff
@@ -472,17 +454,11 @@ constexpr auto kCpplibVer = 0;
 #if _MSC_VER
 #define FOLLY_STORAGE_CONSTEXPR
 #define FOLLY_STORAGE_CPP14_CONSTEXPR
```

Comment:
Let's remove `FOLLY_STORAGE_CONSTEXPR` and let's rename `FOLLY_STORAGE_CPP14_CONSTEXPR` to `FOLLY_STORAGE_CONSTEXPR`.

Looking at this code, it seems that the intention was that icc with c++14 would be okay with `constexpr` storage. AFAICT this just needs to distinguish between gcc/clang (and icc?) v.s. msvc.